src/engine/qcommon/crypto.h

```diff
@@ -27,11 +27,6 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "q_shared.h"
 #include "qcommon.h"
 
-#pragma warning(push)
-#pragma warning(disable : 4146) // "unary minus operator applied to unsigned type, result still unsigned"
-#include <gmp.h>
```

Comment:
The Nettle headers include the GMP header. So this may be here just for the warning fix. You could get rid of the GMP include but the MSVC warning adjustment would still need to be used for the Nettle headers.

Reply:
Ah, I see.